apps/scan/backend/src/app.ts

```diff
@@ -46,11 +49,19 @@ import { getMachineConfig } from './machine_config';
 import { CVR_EXPORT_FORMAT } from './globals';
 import { DefaultMarkThresholds } from './store';
 
-function constructInsertedSmartCardAuthMachineState(
+function constructAuthMachineState(
   workspace: Workspace
 ): InsertedSmartCardAuthMachineState {
   const electionDefinition = workspace.store.getElectionDefinition();
```

Comment:
I see here we don't use `input`; consider standardizing all `constructAuthMachineState` functions to use the same convention.

Reply:
Once nuance here is that VxMark's machine state construction currently looks different than the other machines because it's the one machine where the election definition doesn't yet live on the backend and needs to be provided by the frontend. Once we make that change, VxMark too will just take the `workspace` as its only input